package.json

```diff
@@ -57,7 +57,7 @@
     "renovate-config-standard": "^2.0.0"
   },
   "engines": {
-    "node": "8.* || >= 10.*"
+    "node": "8.3 || >= 10.*"
```

Comment:
Why was locking to 8.3 needed? (This broke a CI build for us running Node v8.16.)

Reply:
oops, that should have been "^8.3". Can you submit a fix?